### PR TITLE
Document Spaces env vars

### PR DESCRIPTION
**Pull request by coyotte508**

### CODE REVIEW

File: docs/hub/spaces-overview.md [hunk 1]
```diff
@@ -97,7 +97,10 @@ Paused time is not billed.
 
 In some cases, you might be interested in having programmatic access to the Space author or repository name. This feature is particularly useful when you expect users to duplicate your Space. To help with this, Spaces exposes different environment variables at runtime. Given a Space [`osanseviero/i-like-flan`](https://huggingface.co/spaces/osanseviero/i-like-flan):
 
+* `CPU_CORES`: 4
+* `MEMORY`: 15Gi
```
Comment (julien-c) [inline docs/hub/spaces-overview.md:101]: they're not namespaced by `SPACE_`?

Comment (julien-c) [reply docs/hub/spaces-overview.md:101]: iguess it makes sense, they're not part of the Space conceptually, but of the runtime 🤷

File: docs/hub/spaces-overview.md [hunk 2]
```diff
@@ -97,7 +97,10 @@ Paused time is not billed.
 
 In some cases, you might be interested in having programmatic access to the Space author or repository name. This feature is particularly useful when you expect users to duplicate your Space. To help with this, Spaces exposes different environment variables at runtime. Given a Space [`osanseviero/i-like-flan`](https://huggingface.co/spaces/osanseviero/i-like-flan):
 
+* `CPU_CORES`: 4
+* `MEMORY`: 15Gi 
 * `SPACE_AUTHOR_NAME`: osanseviero
 * `SPACE_REPO_NAME`: i-like-flan
 * `SPACE_TITLE`: I Like Flan (specified in the README file)
 * `SPACE_ID`: `osanseviero/i-like-flan`
+* `SPACE_SUBDOMAIN`: `osanviero-i-like-flan` (the space's embed url is https://osanviero-i-like-flan.hf.space)
```
Comment (julien-c) [inline docs/hub/spaces-overview.md:106]: nice, i like this one

**Review by julien-c — APPROVED** [review]
lets wait for others approvals but 🔥 lgtm

**Review by christophe-rannou — APPROVED** [review]

**Pull request merged**